rules/Php80/Rector/Class_/AnnotationToAttributeRector.php

```diff
@@ -233,4 +247,13 @@ private function processDoctrineAnnotationClasses(
             );
         }
     }
+
+    private function shouldSkip(PhpDocInfo $phpDocInfo): bool
+    {
+        if ($phpDocInfo->hasByAnnotationClasses(self::SKIP_UNWRAP_ANNOTATIONS)) {
+            return true;
+        }
+
+        return false;
```

Comment:
you should can return directly: 

```suggestion
        return $phpDocInfo->hasByAnnotationClasses(self::SKIP_UNWRAP_ANNOTATIONS);
```

Reply:
fixed